packages/pages/src/bin/spawn.ts

```diff
@@ -6,9 +6,9 @@ import path from "path";
 const __filename = fileURLToPath(import.meta.url);
 const __dirname = path.dirname(__filename);
 
+const filePrefix = path.sep !== path.posix.sep ? "file:\\\\" : "";
 const pathToPagesScript = path.resolve(__dirname, "./pages.js");
-
-const pathToLoader = path.resolve(__dirname, "./loader.js");
+const pathToLoader = filePrefix + path.resolve(__dirname, "./loader.js");
```

Comment:
Doesn't dirname include the Windows drive prefix, like `C:`? (I think it was actually D the Github actions. Is that the correct thing? `file:\\\\D:\\path\to\loader.js`?

Reply:
Yes that is the correct thing. It expects `file:\\\\c:\\path\\etc`